tests/platform_tests/test_auto_negotiation.py

```diff
@@ -439,7 +439,7 @@ def get_cable_supported_speeds(cls, duthost, dut_port_name):
 
         if not cls.device_path:
             cls.device_path = duthost.shell('ls /dev/mst/*_pci_cr0')['stdout'].strip()
-        port_index = cls.sorted_ports[duthost].index(dut_port_name) + 1
+        port_index = cls.get_port_index_for_mlxlink(duthost, dut_port_name)
```

Comment:
@Junchao-Mellanox mlxlink is mellanox specific. is this added under mellanox only?

Reply:
can we use this API to get the port index for a logical port [https://github.com/Azure/sonic-mgmt/blob/master/tests/common/platform/interface_utils.py#L135](https://github.com/Azure/sonic-mgmt/blob/master/tests/common/platform/interface_utils.py#L135) instead of using mellanox specific?